src/nrc/api/serializers.py

```diff
@@ -184,11 +185,17 @@ def _send_to_subs(self, msg: dict):
                 subs.add(group.abonnement)
 
         task_kwargs = {}
-        if settings.LOG_NOTIFICATIONS_IN_DB:
+        if notificatie:
+            task_kwargs.update({"notificatie_id": notificatie.id})
+            task_kwargs.update({"attempt": notificatie.last_attempt + 1})
+
+        # Do not save a new notification, if an existing notification is being scheduled
+        if not notificatie and settings.LOG_NOTIFICATIONS_IN_DB:
             # creation of the notification
             kanaal = Kanaal.objects.get(naam=msg["kanaal"])
             notificatie = Notificatie.objects.create(forwarded_msg=msg, kanaal=kanaal)
             task_kwargs.update({"notificatie_id": notificatie.id})
+            task_kwargs.update({"attempt": notificatie.last_attempt + 1})
```

Comment:
```suggestion
            task_kwargs.update({
                "notificatie_id": notificatie.id,
                "attempt": notificatie.last_attempt + 1,
            })
```

Reply:
I'd also just have the `if notificatie` check below the `if not notificatie and ...` check, so then you only do the `task_kwargs.update` call once, now it's code duplication that is not needed.